src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.Log.cs

```diff
@@ -32,6 +32,9 @@ private static class Log
             private static readonly Action<ILogger, string, Exception> _connectedStarting =
                 LoggerMessage.Define<string>(LogLevel.Debug, new EventId(11, "ConnectedStarting"), "Connection {TransportConnectionId} started.");
 
+            private static readonly Action<ILogger, string, Exception> _migrationStarting =
+                LoggerMessage.Define<string>(LogLevel.Debug, new EventId(22, "MigrationStarting"), "Connection {TransportConnectionId} migrated from another server.");
```

Comment:
why event id is `22`?

Reply:
21 has already been occupied.
And since `connectedStarting` and `migrationStarting` are called in a similar scenario I think they should be put together?

Reply:
interesting, the previous pattern is order by event id though

---
In reply to: [347767989](https://github.com/Azure/azure-signalr/pull/739#discussion_r347767989) [](ancestors = 347767989)

Reply:
Would definitely recommend keeping the IDs in order, otherwise every time you add a new ID you need to check each log to make sure the ID isn't in use.

Reply:
Ok, I'll rearrange it ~